components/code/styles.js

```diff
@@ -0,0 +1,58 @@
+import { createUseStyles } from 'react-jss';
+import { createGenerateId } from '../utils/helpers';
+
+const colors = {
+  gray1: '#E5E5E5',
+  gray2: '#6D6D6D',
+  gray3: '#1D1D1F',
+  orange1: '#FFB45B',
+};
+
+const useStyles = createUseStyles({
```

Comment:
Codacy has a fix for the issue: [Expected newline after '('.](https://app.codacy.com/gh/melishev/wuki/pullRequest?prid=8586480)
```suggestion
const useStyles = createUseStyles(
```